typing/env.ml

```diff
@@ -150,7 +150,7 @@ end  = struct
 
 end
 
-module PathMap = Map.Make(Path)
+module NameMap = Misc.StringMap
```

Comment:
Names of what?

Reply:
`Name`, as in the individual parts in an `Longindent.t`. I would like to call them ident, but that clashes with the individual parts of a `Path.t`. I insist on having a local redefinition on that one, as it's the original motivation for this whole patch. But I agree, the name is bad.